src/core/VarReplay.jl

```diff
@@ -11,7 +11,9 @@ import Base: string, isequal, ==, hash, getindex, setindex!, push!, show, isempt
 import Turing: link, invlink
```

Comment:
Consider renaming `VarReplay.jl` to `VarInfo.jl`, and renamaing submodule `VarReplay` to `VarInfo`.

Reply:
Can we call the module `Parameters`, `ParametersInfo`, `Variables`, `VariablesInfo` or `VarRecords` instead? Any other names are welcome too. `VarInfo` is used to construct `UntypedVarInfo` using `VarInfo()`, and will be used to construct `TypedVarInfo` using `VarInfo(m::Model)`. I like `VarInfo` for the constructor because it is short and we are used to it.

Reply:
Shall we call it `RandomVariables` (or `RanVars` if you want something mysterious)? This makes the purpose of the submodule clearer than `VarInfo` actually.

Reply:
I'm up to `RandomVariables`!

Reply:
If no objections happen or better suggestions get made, I will make the change tomorrow.